File: 240405/아래로 사각형 채우기/filling-rectangle-with-downflow.py
n = int(input())
cnt = 1

arr2 = [
    [0 for _ in range(n)]
    for _ in range(n)
]

for i in range(n):
    for j in range(n):
        arr2[j][i]=cnt
        cnt+=1

for row in arr2:
    for i in row:
        print(i,end=" ")
    print()

File: 240626/아래로 사각형 채우기/filling-rectangle-with-downflow.js
const fs = require("fs")
let input = fs.readFileSync(0).toString().trim()

let n = Number(input)

let arr = Array(n).fill(0).map(()=>Array(n).fill(0))

let cnt = 1

for(let i = 0; i<n; i++){
    for(let j =0; j<n; j++){
        arr[j][i] = cnt;
        cnt++
    }
}


let str =""

for(let i=0; i<n; i++){
    str = ""
    for(let j=0; j<n; j++){
        str+=arr[i][j]+" "
    }
    console.log(str)
}